sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Added support for service version 2021-10-04.
 
+### Bugs Fixed
+
+- Fixed an issue of under some conditions, it may not be able to correctly parse URL with account name in URL path.
```

Comment:
What is the `it`?

Reply:
.Net can get storage account name, container name, blob name from a URL like: "https://<domain>:10001/<accountname>/<containername>/<blobname>", while JS would recognize <accountname> as container name.

When there's an available azure port in host part, .Net would treat the first path segment as account name.  This PR is to keep consistency with .Net.